recipes/podofo/all/conanfile.py

```diff
@@ -36,7 +36,7 @@ class PodofoConan(ConanFile):
         "threadsafe": True,
         "with_openssl": True,
         "with_libidn": True,
-        "with_jpeg": True,
```

Comment:
This would be a breaking change for users already setting `&:with_jpeg=True/False`, as the option won't exist anymore.

The proper way to move forward is to keep the old option name,
and have its possible values be `[True, False, "libjpeg", "libjpeg-turbo", "mozjpeg"]`,
where being `True` would be an alias for `libjpeg`, something like:

```py
@property
def _with_jpeg(self):
    return {True: "libjpeg"}.get(self.options.with_jpeg, self.options.with_jpeg)
```

(Or a variation of it as you see fit!)